### DISCUSSION TITLE
Ceph+Cloudstack for HCI

### DISCUSSION THREAD

**Discussion by gobiursus** [apache/cloudstack#9865] [Q&A] [answered]
##### ISSUE TYPE

 * Documentation Report

##### COMPONENT NAME

~~~
Documentation
~~~

##### CLOUDSTACK VERSION

~~~
All versions
~~~

##### OS / ENVIRONMENT

Debian 12.7 Stable

##### SUMMARY

Does Cloudstack support Hyperconverged setup? Like multiple All-In-One nodes with Ceph for storage. Can we document it?
Thank you!

**Comment by rajujith** [comment, marked as answer]
@gobiursus  Please check https://github.com/apache/cloudstack/discussions/9363
